Assignment2/bgDani-DanielBerrocal.ttl

```diff
@@ -1,10 +1,10 @@
 @base <http://www.oeg-upm.net/Organization/>
-@prefix pre: <http:/www.oeg-upm.net/Organization#/>
+@prefix pre: <http://www.oeg-upm.net/Organization#>
```

Comment:
Good point, I had not realised. The first one was also possible with the final hash code. We will talk about it when talking about Linked Data